app/models/organization.rb

```diff
@@ -56,6 +56,35 @@ def one_owner_remains?
     users.count == 1
   end
 
+  def organization_webhook_active?
+    return false if webhook_id.nil?
+
+    begin
+      github_client.org_hook(github_id, webhook_id).active
+    rescue Octokit::NotFound
+      false
+    end
+  end
+
+  def create_organization_webhook
```

Comment:
@tarebyte this code is duplicated from the `Organization::Creator` class. Do you have any suggestions on how to move it into one place? It's locked up in the creator and not callable from anywhere else.